blockbase/theme.json

```diff
@@ -198,6 +198,9 @@
 					}
 				}
 			},
+			"layout": {
+				"contentSize": "620px"
```

Comment:
We should probably also look into making it so the layout key generates a CSS variable, because this is redundant.

Reply:
Couldn't find anything so opened this: https://github.com/WordPress/gutenberg/issues/33705